sdk/examples/BladeExample.cpp

```diff
@@ -17,27 +17,73 @@
  * limitations under the License.
```

Comment:
Before this merges, I would change the name of the file to describe the example, rather than referencing a particular end user

Reply:
Yeah I just made this a draft to utilize the GitHub windows build, I'll make it more official when I make this an official PR